sql-ballerina/Module.md

```diff
@@ -0,0 +1,302 @@
+## Overview
+
+This module provides the common interface and functionality to interact with a database. The corresponding database
+clients can be created by using specific database modules such as `MySQL` or using the Java Database Connectivity 
+module `JDBC`.
+
+### List of Database Modules
+Ballerina now has [JDBC module](https://ballerina.io/learn/api-docs/ballerina/#/jdbc) as the generic db connector module to connect with any relational database by simply providing the JDBC URL and the other related properties.
+Ballerina also provides specially designed various database specific db connectors, so that you can work with different databases, and you are allowed to access their db specific functionalities.
+
+### Client
+
+The database client should be created using any of the above-listed database modules and once it is created, the operations and functionality explained below can be used. 
+
+#### Connection Pool Handling
+
+All database modules share the same connection pooling concept and there are 3 possible scenarios for 
+connection pool handling.  For its properties and possible values, see the `sql:ConnectionPool`.  
+
+1. Global shareable default connection pool
+
+    If you do not provide the `poolOptions` field when creating the database client, a globally-shareable pool will be 
+    created for your database unless a connection pool matching with the properties you provided already exists. 
+    The JDBC module example below shows how the global connection pool is used. 
+
+    ```ballerina
+    jdbc:Client|sql:Error dbClient = 
+                               new ("jdbc:mysql://localhost:3306/testdb", 
+                                "root", "root");
+    ```
+
+2. Client owned, unsharable connection pool
```

Comment:
```suggestion
2. Client-owned, unsharable connection pool
```

Reply:
Change Added in https://github.com/ballerina-platform/module-ballerina-sql/pull/228/commits/2656f12c62276e15a256c133465cb60304d0b041